blocks/blog-home/blog-home.css

```diff
@@ -58,11 +58,16 @@
 
 .blog-home[data-block-status="loaded"] > .blog-card > .card-content p {
   font-size: 18px;
-  font-family: 'GFS Didot', serif;
+  font-family: 'GFS Didot', Times, serif;
   font-style: italic;
   color: var(--text-color);
 }
 
+@font-face {
+  font-family: serif;
```

Comment:
@helms-charity Yes, on the blog home page there is a big layout shift because on the cards because of the font used in those cards , that font face indicates clearly which is the fallback font that needs to be used when in serif, if not the font used by default can also generate layout shift and impact the LHS